src/Command/CodeStudio/CodeStudioChangePHPVersionCommand.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Acquia\Cli\Command\CodeStudio;
+
+use Acquia\Cli\Command\CommandBase;
+use Gitlab\Exception\RuntimeException;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class CodeStudioChangePHPVersionCommand extends CommandBase {
+
+  use CodeStudioCommandTrait;
+
+  protected static $defaultName = 'codestudio:update-php-version';
+
+  /**
+   * {inheritdoc}.
+   */
+  protected function configure(): void {
+    $this->setDescription('Change the PHP version in Code Studio')
+      ->addArgument('php-version', InputArgument::REQUIRED, 'The PHP version that needs to configured or updated')
+      ->addOption('gitlab-token', NULL, InputOption::VALUE_REQUIRED, 'The GitLab personal access token that will be used to communicate with the GitLab instance')
+      ->addOption('gitlab-host-name', NULL, InputOption::VALUE_REQUIRED, 'The GitLab hostname.');
+    $this->acceptApplicationUuid();
+  }
+
+  /**
+   * @param \Symfony\Component\Console\Input\InputInterface $input
+   * @param \Symfony\Component\Console\Output\OutputInterface $output
+   *
+   * @return int
+   * @throws \Exception
+   */
+  protected function execute(InputInterface $input, OutputInterface $output): int {
+    $php_version = $input->getArgument('php-version');
```

Comment:
Validate the PHP version. Use https://github.com/acquia/cli/blob/83dfb30525e7f7bea326c3cc53b00566f2474d68/src/Command/Ide/IdePhpVersionCommand.php#L87 . Move that method out of the IdeCommandTrait to somewhere more common like the command base, and pull the last line that's specific to IDEs.